### ISSUE TITLE
Jaws can't fully interact with keyboard

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/number-pairs#147]
**Device**
Samsung Laptop
**OS**
Win 11
**Browser**
Chrome
**Problem Description**
For https://github.com/phetsims/qa/issues/1219
When trying to use the arrow keys to move the beads while Jaws is running, the beads only move slightly, or not at all. I was also unable to toggle the kittens with the keyboard.
**Visuals**

https://github.com/user-attachments/assets/eb264713-0992-4969-a2f3-0d7a8403f508

**Comment by Matthew-Moore240** [comment]
When using NVDA, I have no issue moving the beads, but trying to do anything with the kittens doesn't work.

**example**

https://github.com/user-attachments/assets/407cab5d-0f8f-4078-baa4-f9564fd53cf3

**Comment by jessegreenberg** [comment]
The JAWS issue is a screen reader bug unfortunately. We submitted a bug report to Freedom Scientific about it earlier this year https://github.com/phetsims/a11y-research/issues/9.

For the kittens it is likely that the draggable Node requires the application role. Around here:
https://github.com/phetsims/number-pairs/blob/a8a32b330a8f2fcf38b782286c5b09fe5af74a0d/js/common/view/KittenNode.ts#L72-L76

We have `AccessibleDraggableOptions` that you can apply with `combineOptions`. I am also wondering if a new mixin would be better for these.

**Comment by KatieWoe** [comment]
Does that mean that part of the sim is unusable with Jaws?

**Comment by jessegreenberg** [comment]
One way around that could be to use dragDelta instead of dragSpeed. Then the bead/kitten will move at least a little bit per press. Ill talk to @marlitas about that when we meet tomorrow.

**Comment by marlitas** [comment]
I talked to @jessegreenberg about this a bit more, and it seems like a design decision in the end. For a better experience with JAWS we should use dragDelta, however the drag experience on keyboard in general will not be as smooth as it currently is. It will be a bit more jumpy. @catherinecarter and I should talk through this more to decide how to move forward.

**Comment by catherinecarter** [comment]
I looked at trig tour, and while more obvious that the movement isn't as fluid, the non-fluid con seems insignificant compared to the benefit of users being able to interact with the sim. So let's change the movement to be `dragDelta` so any user needing JAWS can use the sim.

**Comment by Matthew-Moore240** [comment]
The issue for NVDA is still present. Still can't toggle the kittens or move them. 

**Clip**
Tried using Space Bar and Enter keys to toggle the kittens, and I tried moving them at all with the arrow keys 

https://github.com/user-attachments/assets/0dcc33ab-dfd6-42ae-923d-4fe87bdac622

**Comment by KatieWoe** [comment]
Agreed. The beads are working much better, but I'm seeing the same issues with the kittens with JAWS

**Comment by jessegreenberg** [comment]
Sorry @marlitas , that probably still requires this part: https://github.com/phetsims/number-pairs/issues/147#issuecomment-2755178636

**Comment by marlitas** [comment]
Oh my gosh yes... So sorry we talked about that and I totally forgot. @Matthew-Moore240 can you check the kittens again? That should be fixed now.

**Comment by Matthew-Moore240** [comment]
@marlitas This seems like a big improvement. The kittens can be toggled now, and they move, but the movement is a little odd. 

**Clip** 

- Holding down an arrow works well. 
- There seems to be some time needed in between each arrow key click for the kitten to move. 
- Rapid arrow key clicking makes a sound, but the kitten doesn't move  

https://github.com/user-attachments/assets/e5011260-aa5c-4bb6-8b2c-9e3e005fbfbd

**Comment by jessegreenberg** [comment]
@Matthew-Moore240 what platform were you using for that recording?

**Comment by Matthew-Moore240** [comment]
@jessegreenberg i'm using the Windows snipping tool in the above clip. Here's a new video recorded with OBS(Streamlabs desktop version)   

**Clip**

https://github.com/user-attachments/assets/fe59e938-9bea-41af-a5b5-4e2eb858d56b

**Comment by jessegreenberg** [comment]
Thanks @Matthew-Moore240 - 

I just tested on Windows 10 with JAWS 2025 and cannot reproduce either of these items:
> There seems to be some time needed in between each arrow key click for the kitten to move.
> Rapid arrow key clicking makes a sound, but the kitten doesn't move

@KatieWoe or @Nancy-Salpepi can you please try to reproduce this problem?

**Comment by KatieWoe** [comment]
I didn't see these issues on Main either. I did see https://github.com/phetsims/number-pairs/issues/148 was seemingly back?

**Comment by KatieWoe** [comment]
I mean it was reading 50, 49, etc. on Jaws again

**Comment by marlitas** [comment]
Okay I believe the weird kitten dragging is fixed above @Matthew-Moore240. Can you check?

@KatieWoe I'm checking in with JG on the different results between Jaws and VoiceOver... I'm not sure of the technical differences there and why they might be producing two different results.

**Comment by Matthew-Moore240** [comment]
@marlitas Looks good on my end!

**Comment by marlitas** [comment]
Okay great. I think the main issue here has been addressed and the remaining work is covered by https://github.com/phetsims/number-pairs/issues/148. If anyone disagrees or if I missed something please reopen.
